hr_timesheet_purchase_order/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+* Add a *Billing partner* in the *HR* tab of the *Employee* form and enable the *Purchase Order* checkbox
```

Comment:
* Go to *_Employees_* app > select an employee > go to *_HR Settings_* tab and enable the *_Generate Purchase Order from timesheet sheet_" checkbox
* Select the *_Billing partner_* for which the PO will be created